.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: HACF Discord
+    url: https://discord.gg/PaZFEjX
+    about: The HACF discord server
+  - name: HACF Forums
+    url: https://forum.hacf.fr/
+    about: The HACF forum
```

Comment:
To ensure passing all CI checks, you need to add an empty newline missing at the end of the file.